apis/config/v1beta1/configuration_types.go

```diff
@@ -339,6 +340,9 @@ type Integrations struct {
 	// the expected format is `Kind.version.group.com`.
 	ExternalFrameworks []string `json:"externalFrameworks,omitempty"`
 	// PodOptions defines kueue controller behaviour for pod objects
+	// Deprecated: This field will be removed on v1beta2, use ManagedJobsNamespaceSelector
+	// (https://kueue.sigs.k8s.io/docs/tasks/run/plain_pods/)
```

Comment:
```suggestion
	// (https://kueue.sigs.k8s.io/docs/tasks/manage/enforce_job_management/manage_jobs_without_queue_name/)
```

I think this is more eligible for ManagedJobsNamespaceSelector documentation.

Reply:
That was my initial link here, but it mainly talks about using it in conjunction with `manageJobsWithoutQueueName`. The plain pods link has more relevant content that talks about how to use `managedJobsNamespaceSelector` with the pod integration.  So I thought it would be a better link for users who had to migrate.

Reply:
That sounds good. Actually, `ManagedJobNamespaceSelector` is not only for Pod integration. So it would be nice to restructure https://kueue.sigs.k8s.io/docs/tasks/manage/enforce_job_management/manage_jobs_without_queue_name/ so that we can explain how to enforce and configure those behaviors.

Anyway, that is out of the scope of this PR.